modules/HardwareDrivers/PowerSupplies/Huawei_V100R023C10/fusion_charger_lib/goose-lib/CMakeLists.txt

```
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.22)
```

```suggestion
cmake_minimum_required(VERSION 3.16)
```
https://github.com/EVerest/everest-core/pull/1336#discussion_r2309666120 still applies